docs/source/developer/xkcd_extension_tutorial.rst

```diff
@@ -748,13 +748,9 @@ Finally, rewrite the ``activate`` function so that it:
       // Add the command to the palette.
       palette.addItem({ command, category: 'Tutorial' });
 
-      // Track and restore the widget state
-      let tracker = new InstanceTracker<Widget>({ namespace: 'xkcd' });
-      restorer.restore(tracker, {
-        command,
-        args: () => JSONExt.emptyObject,
-        name: () => 'xkcd'
-      });
+      // Track and restore the widget state.
+      let tracker = new WidgetTracker({ namespace: 'xkcd' });
+      void restorer.restore(tracker, { command, name: () => 'xkcd' });
```

Comment:
If @jaipreet-s's work is ready, I'm happy to handle the merge conflict. Otherwise, the two things that have changed are:

* `tracker` is now an `InstanceTracker`.
* `restorer.restore(...)` no longer requires an `args` field; it is optional.

Reply:
I don't think it's a problem, just flagging it for him.